src/test-utils/render.js

```diff
@@ -1,18 +1,24 @@
 /* eslint-disable react/prop-types */
+import { DataProvider } from '@dhis2/app-runtime'
 import { render as renderOrig } from '@testing-library/react'
 import React from 'react'
-import { QueryClient } from 'react-query'
 import { MemoryRouter } from 'react-router-dom'
 import { OuterComponents } from '../app/app-wrapper.js'
+import { useTestQueryClient } from './index.js'
 
 export function TestWrapper({
     wrapper: WrapperFromOptions,
     router = MemoryRouter,
     queryClient,
     children,
 }) {
+    const defaultTestQueryClient = useTestQueryClient()
```

Comment:
since I have to use a hook under the hood (`useDataEngine()`), then this needs to live in a React component so made it the default here. Not sure if there is a non-hook way to get hold of the data engine?

Reply:
> Not sure if there is a non-hook way to get hold of the data engine?

Unfortunately there isn't.. 
I'm OK with the change, please import `useTestQueryClient` from `'./use-test-query-client.js`, not from the `index.js`, which is the entrypoint to this "namespace" from the outside.

Reply:
> I'm OK with the change, please import useTestQueryClient from './use-test-query-client.js, not from the index.js, which is the entrypoint to this "namespace" from the outside.

done